packages/venia-ui/lib/components/MiniCart/__tests__/__snapshots__/cartOptions.spec.js.snap

```diff
@@ -1,7 +1,7 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`it renders the correct tree 1`] = `
-<t>
+<Q>
```

Comment:
I genuinely don't know why this changed but I imagine it has something to do with informed.